lib/string_reverse.rb

```diff
@@ -1,7 +1,7 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
   return nil if my_string == nil
-  
+
   front_index = 0
   back_index = my_string.length - 1
 

```

Comment:
Minor: You are currently using two temporary variables, `front_value` and `back_value`. Technically, you could optimize by using just one temporary value. Whilst this won't change the algorithmic design complexity, it will make more concise code.
e.g.
```ruby
while front_index < back_index
   front_value = my_string[front_index]
   my_string[front_index] = my_string[back_index]
   my_string[back_index] = front_value

   front_index += 1
   back_index -= 1
end
```